src/join-7.sol

```diff
@@ -88,6 +103,7 @@ contract GemJoin7 {
         uint256 wad = mul(sub(gem.balanceOf(address(this)), bal), 10 ** (18 - dec));
         require(int256(wad) >= 0, "GemJoin7/overflow");
         vat.slip(ilk, urn, int256(wad));
+        emit Join(urn, wad);
```

Comment:
We probably want to log amt and not wad here. For example in GemJoin8 we log the input parameter and not the decimalss scaled quantity.  Also in exit() in this GemJoin.

Reply:
Yeah good point. I was going by the slip amounts elsewhere but I agree collateral units are better.